freeipa/src/main/java/com/sequenceiq/freeipa/entity/Stack.java

```diff
@@ -35,6 +35,8 @@
 
     private String environment;
 
+    private String tenant;
```

Comment:
do we need to adjust the @Table annotation at the top to add the tenant tot he @UniqueConstraint columnNames?

Reply:
Good point, adjusted.